packages/validator/src/util/clock.ts

```diff
@@ -81,6 +82,39 @@ export class Clock implements IClock {
     return Date.now() / 1000 - (this.genesisTime + this.config.SECONDS_PER_SLOT * slot);
   }
 
+  msToAttestationDue(slot: number): number {
```

Comment:
do we really need those? need to maintain them separately as they are no reusing other helpers

there also doesn't seem to be a reason to calculate slot fraction, eg. in attestation service can just do
```ts
await Promise.race([sleep(this.clock.msToSlot(slot) + getAttestationDueMs(this.config), signal), this.emitter.waitForBlockSlot(slot)]);
```

or we add a `msFromSlot` helper